test/t8_gtest_version.cxx

```diff
@@ -42,7 +42,7 @@ TEST (t8_gtest_version, major_version)
 TEST (t8_gtest_version, minor_version)
 {
   /* Change this number when you increase the minor version. */
-  const int minor_version = 4;
+  const int minor_version = 5;
```

Comment:
In line 30 we mention the current version number, too. Please change it to 1.5.0 too.